testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -180,6 +180,11 @@ class TestAtomnames(TestAtomAttr):
     single_value = 'Ca2'
     attrclass = tpattrs.Atomnames
 
+class TestemptyAtomnames(TestAtomAttr): 
```

Comment:
A class seems a little overkill for this simple test, especially as it subclasses `TestAtomAttr` (and thereby runs all the tests defined there again). Could you just make this a function?

These should also be two separate tests, I think, as they are different methods.

Reply:
Also, the name of this class is a little misleading -- you're testing empty residue groups, and the contents of the atom names has fairly little to do with it. In fact, as the methods are transplanted to `ResidueGroup`, a more concise test could just look at what ``u.residues[[]]._get_next_residues_by_resid()`` returns.

Reply:
Hi, Thanks for the recommendations! I'll cut it down a bit!

Reply:
Just wanted to let you know that I changed the class to functions in my latest commit. Hopefully, that is what you were looking for ?

Reply:
Thank you!